File: src/main/java/catwalks/util/meta/MetaMapProperty.java
package catwalks.util.meta;

/**
 * Created by TheCodeWarrior
 */
public abstract class MetaMapProperty<K, V> extends MetaProperty {
	public abstract void set(MetaStorage storage, K key, V value);
	public abstract V get(MetaStorage storage, K key);
	
}

File: src/main/java/catwalks/util/meta/MetaMapProperty.kt
package catwalks.util.meta

/**
 * Created by TheCodeWarrior
 */
abstract class MetaMapProperty<K, V> : MetaProperty() {
    abstract operator fun set(storage: MetaStorage, key: K, value: V)
    abstract operator fun get(storage: MetaStorage, key: K): V

}
